wasmcloud-test-util/src/provider_test.rs

```diff
@@ -462,12 +466,12 @@ macro_rules! run_selected_spawn {
 // calls that fail.
 pub async fn run_tests(
     tests: Vec<(&'static str, TestFunc)>,
-) -> std::result::Result<Vec<TestResult>, Box<dyn std::error::Error>> {
-    let mut results: Vec<TestResult> = Vec::new();
+) -> std::result::Result<Vec<crate::testing::TestResult>, Box<dyn std::error::Error>> {
```

Comment:
This could be made a bit more readable by bringing `crate::testing::TestResult` into scope

Reply:
There's a poorly documented reason for doing that - there are two different TestResult symbols that come from different namespaces, and I thought that by using the full path to the symbol, the distinction would be clearer, but it's too subtle and needs to be documented explicitly. I'll address both issues in the next rev. Thanks for the review!